geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/RebalanceCommand.java

```diff
@@ -105,6 +105,12 @@ public ResultModel rebalance(
     return result;
   }
 
+  public Callable<ResultModel> rebalanceCallable(String[] includeRegions, String[] excludeRegions,
+      boolean simulate) {
+    return new ExecuteRebalanceWithTimeout(includeRegions, excludeRegions, simulate,
```

Comment:
It seems like rebalanceCallable or ExecuteRebalanceWithTimeout should make sure to check the ResultModel it is returning and if it has a single error (see line 102) to set the status to ERROR (line 103).

Reply:
I pulled this common code inside of the callable.